.coderabbit.yml

```
@@ -11,5 +11,78 @@ reviews:
+            prev: '*'
+            next: return
+          - blankLine: always
+            prev: ['const', 'let', 'var']
+            next: '*'
+          - blankLine: any
+            prev: ['const', 'let', 'var']
+            next: ['const', 'let', 'var']
+
+        # React specific
+        react/jsx-pascal-case: error
+        react/jsx-filename-extension: [error, { extensions: ['.tsx', '.jsx'] }]
```

yamllint (1.35.1)

[error] too many spaces inside braces

(braces)